cookbooks/aws-parallelcluster-environment/templates/cfn_hup_configuration/ComputeFleet/cfn-hup-update-action.sh.erb

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+set -ex
+
```

Comment:
Could you please add a comment explaining what the script is supposed to do and how it should be invoked?

Reply:
Will do!